gapis/shadertools/shadertools.go

```diff
@@ -189,6 +189,35 @@ func descriptorBindingLess(a DescriptorBinding, b DescriptorBinding) bool {
 	return a.SpirvId < b.SpirvId
 }
 
+func ParseSourceCode(shader []uint32) (string, error) {
```

Comment:
This name is misleading/inaccurate. I think something like `ExtractDebugSource` would be better.

Reply:
I'm also going to put the SPIRV-Cross logic in this function. Do you recommend I put it somewhere else?

Reply:
Note: I use SPIRV-Reflect to set some of the compiler options, like version number, for SPIRV-Cross